packages/core/src/utils/multipleTemplates.ts

```diff
@@ -135,6 +135,25 @@ export function normalizePDPTemplate(templateValue: string) {
   return formattedValue.toLowerCase()
 }
 
+// Returns an array of slugs without the number at the end ("-{number}/p") at each interaction if it matches, otherwise returns the slug as is.
+const getSlugsWithoutSkuIdFromPDP = (slug: string) => {
+  const slugs = []
+  let currentSlug = slug
+
+  while (true) {
+    const match = currentSlug.match(/-\d+\/p$/) // Matches the pattern "-{number}/p" at the end of the slug
+    if (match) {
+      const newSlug = currentSlug.replace(/-\d+\/p$/, '/p') // Replace the last number before '/p'
+      slugs.push(newSlug)
+      currentSlug = newSlug
+    } else {
+      break
+    }
```

Comment:
Is there any case the current slug doesn't match and there's still another one to be processed later? Because this `break` clause would finish the process.

p.s: This code suggestion isn't related to the question above
```suggestion
    }

    break
```

Reply:
Good catch, but this case will not occur heheh
I updated this method to improve readability as well; I'll ask for the review again.